pkg/controller/node/nodecontroller_test.go

```diff
@@ -451,6 +455,67 @@ func TestMonitorNodeStatusEvictPods(t *testing.T) {
 	}
 }
 
+// TestCloudProviderNoRateLimit tests that monitorNodes() immediately deletes
+// pods and the node when kubelet has not reported, and the cloudprovider says
+// the node is gone.
+func TestCloudProviderNoRateLimit(t *testing.T) {
+	tmp := nodeExistsInCloudProvider
```

Comment:
Please don't test things like this. If you absolutely must, you *have* to restore the value in a defer block (because t.Failf exits the test immediately).